docker-compose/integrator-analytics/README.md

```diff
@@ -4,39 +4,41 @@
 
 ## Prerequisites
 
-  * [Docker](https://www.docker.com/get-docker) and [Docker Compose](https://docs.docker.com/compose/install/#install-compose) are required to run this deployment.
-
+ * [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Docker](https://www.docker.com/get-docker) and [Docker Compose](https://docs.docker.com/compose/install/#install-compose) are required for running this Docker Compose template.
+ * In order to run the Docker Compose setup, you will need an active subscription from WSO2 since the 
+   Docker images hosted at docker.wso2.com contains the latest updates and fixes to WSO2 Enterprise Integrator. 
+   You can sign up for a Free Trial Subscription [here](https://wso2.com/free-trial-subscription).
+ * If you wish to run the Docker Compose setup using Docker images built locally, build Integrator, Analytics Docker images using [Dockerfiles](../../README.md) and remove the `docker.wso2.com/` prefix from the `image` name In the `docker-compose.yml`. 
```

Comment:
Shall we change `(../../README.md)` to `(../../dockerfiles/README.md)` ? That is point to the readme in `dockerfiles` dir.